src/resources/js/components/ListPresentations.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <div class="container">
+    <div v-if="showMessage">
+      <h3>{{ message }}</h3>
+    </div>
+
+    <div v-for="presentation in presentations.data" :key="presentation.id">
+      <router-link
+        :to="{ name : 'presentation', params : { idPresentation : presentation.id} }"
+        >
+        {{ presentation.title }}
+      </router-link>
+    </div>
+
+    <router-view></router-view>
+  </div>
+</template>
+
+<script>
+
+import axios from 'axios';
+
+export default {
+  name: 'ListPresentations',
+  props: ['parent'],
+  data() {
+    return {
+      presentations: [],
+      showMessage: false,
+      message: '',
+    };
+  },
+  beforeMount() {
+    const apiUrl = `/api/v1/users/${this.parent.loggedUser.id}/presentations`;
+
+    axios
+      .get(apiUrl)
+      .then((response) => {
+        this.presentations = response.data;
+        if (this.presentationsIsEmpty()) {
+          this.showMessage = true;
+          this.message = 'you haven\'t subscribed to a presentation';
+        }
+      })
+      .catch((error) => {
+        this.showMessage = true;
+        this.message = `error while sending data, code ${error.response.status}`;
```

Comment:
Error messages should be in french

Reply:
right translated in [8b643](https://github.com/kayoumido/HEIGVD-PRO-A-11/pull/115/commits/8b643f2f8162c919f22f5521070fdd61f4de5236)